desktop/core/src/desktop/js/jquery/plugins/jquery.filechooser.js

```diff
@@ -810,6 +810,18 @@ Plugin.prototype.init = function () {
       '<div class="filechooser-container" style="position: relative"><div class="filechooser-services" style="position: absolute"></div><div class="filechooser-tree" style="width: 560px"></div></div>'
     );
   $.post('/filebrowser/api/get_filesystems', data => {
+    var initialHome = '/';
+    if (self.options.initialPath == '') {
+      if (Object.keys(data["filesystems"]).length > 1 && Object.keys(data["filesystems"]).indexOf('hdfs') >= 0){
```

Comment:
No need to adjust as I think it's better to switch this one over to Vue soon, so more of an FYI:

- In js you can do `data.filesystems` instead of `data["filesystems"]`
- Could simplify by doing `var filesystems = Object.keys(data.filesystems);` before the first `if`.
- `!= -1` is a more common check for existence than `>= 0`